cloudinstall/multi_install.py

```diff
@@ -224,7 +224,7 @@ def add_bootstrap_to_no_proxy(self):
         if out['status'] != 0:
             log.debug("error from get-env: {}".format(out))
             raise Exception("Problem getting existing no-proxy setting")
-        no_proxy = "{},{}".format(out['output'], bootstrap_ip)
+        no_proxy = "{},{}".format(out['output'].replace('\n', ''), bootstrap_ip)
```

Comment:
I think we need to add a param to get_command_output to change the 'universal_newlines' param for the Popen obj that it calls.

Just replacing newlines with '' could eventually result in invalid YAML in the earlier call to 'juju status', and fixing that will require the newlines change, so we might as well fix this one that way too